ytmusicapi/parsers/playlists.py

```diff
@@ -38,8 +38,10 @@ def parse_playlist_header(response: Dict) -> Dict[str, Any]:
         playlist["duration"] = (
             None if not has_duration else second_subtitle_runs[has_views + has_duration]["text"]
         )
-        song_count = second_subtitle_runs[has_views + 0]["text"].split(" ")
-        song_count = to_int(song_count[0]) if len(song_count) > 1 else 0
+        song_count_text = second_subtitle_runs[has_views + 0]["text"]
+        song_count_search = re.search(r"\d+", song_count_text)
```

Comment:
Thanks for the contribution. Could you please add a test to check that it works for all supported languages?

Not sure what oddities we might be missing here.